src/components/Details/VmDetails/fixtures/VmDetails.fixture.js

```diff
@@ -23,7 +23,27 @@ export const vmFixtures = {
   },
   runningVm: {
     metadata,
-    spec: { running: true },
+    spec: {
+      template: {
+        spec: {
+          domain: {
+            devices: {
+              rng: {},
+              interfaces: [
+                { bridge: {}, name: 'podNetworkName', bootOrder: 2 },
+                { bridge: {}, name: 'pxeNetworkName', bootOrder: 1 },
+              ],
+              disks: [
+                { disk: {}, name: 'disk-one', bootOrder: 3 },
```

Comment:
can you add a cloud init to the vm ? so that we can see the snapshot does not contain it

Reply:
Done!